apps/metadata-editor-e2e/src/e2e/edit.cy.ts

```diff
@@ -57,4 +57,19 @@ describe('editor form', () => {
     })
     cy.get('md-editor-publish-button').click()
   })
+
+  describe.only('date range in sortable list', () => {
```

Comment:
@jahow  should we add a linter rule to prevent `only` in tests? ^^'

Reply:
oops!!! yes that would be great!